setup.py

```diff
@@ -93,7 +93,17 @@
         numpy_include = np.get_numpy_include()
 
     # Determine if we have an available BLAS implementation
-    if 'bdist_wheel' not in sys.argv:
+    if platform.system() == 'Windows':
```

Comment:
Don't you think we should do this on all platforms even if the bug currently only occurs on Windows?

Reply:
finding BLAS seems to work ok on Linux - i.e. it only adds the defines when there's one installed, at least in my experience

Reply:
I agree but if for any reason `check_cblas.py` failed, wouldn't this mean that we should set `blas_info` to `{}`, no matter on which platform this happens?

Reply:
```check_cblas.py``` is only a compilation check, and distutils might do some other magic to add blas arguments to the compile/link commands

https://github.com/X-DataInitiative/tick/blob/master/setup.py#L97 

if we see issues on other platforms it's not difficult to include the compilation there too

Reply:
Alright, I was not thinking about distutils magic, I thought `check_cblas.py` would necessarily imply setup.py to fail